Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -227,6 +228,37 @@ void MenuBar::AddToolsMenu()
   tools_menu->addAction(tr("Start &NetPlay..."), this, &MenuBar::StartNetPlay);
   tools_menu->addAction(tr("Browse &NetPlay Sessions...."), this, &MenuBar::BrowseNetPlay);
 
+#ifdef USE_RETRO_ACHIEVEMENTS
+#ifdef ENABLE_RAINTEGRATION
+  tools_menu->addSeparator();
+
+  if (AchievementManager::GetInstance()->IsDLLEnabled())
+  {
+    QMenu* ra_dev_ipl = tools_menu->addMenu(tr("Achievement Development"));
+
+    const auto items = AchievementManager::GetInstance()->GetMenuItems();
+    for (const auto& [id, title, checked] : items)
+    {
+      if (id == 0)
+      {
+        ra_dev_ipl->addSeparator();
+        continue;
+      }
+
+      QAction* raAction = ra_dev_ipl->addAction(QString::fromUtf8(title));
+      if (checked)
+      {
+        raAction->setCheckable(true);
+        raAction->setChecked(checked);
+      }
+
+      connect(raAction, &QAction::triggered, this,
+              [this, id = id]() { emit ActivateRAMenuItem(id); });
```

Comment:
```suggestion
              [this, id]() { emit ActivateRAMenuItem(id); });
```

should be sufficient, since it captures it by value rather than by reference